README.md

```diff
@@ -1,29 +1,56 @@
-- an `OWNERS` with the project leads listed as approvers ([docs on `OWNERS` files][owners])
-- a `CONTRIBUTING.md` outlining how to contribute to the project
-- an unmodified copy of `code-of-conduct.md` from this repo, which outlines community behavior and the consequences of breaking the code
-- a `LICENSE` which must be Apache 2.0 for code projects, or [Creative Commons 4.0] for documentation repositories, without any custom content
-- a `SECURITY_CONTACTS` with the contact points for the Product Security Team
-  to reach out to for triaging and handling of incoming issues. They must agree to abide by the
-  [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
-  and will be removed and replaced if they violate that agreement.
+### Resources
 
-## Community, discussion, contribution, and support
+* [design doc](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/20190610-machine-states-preboot-bootstrapping.md)
+* [cluster-api.sigs.k8s.io](https://cluster-api.sigs.k8s.io)
+* [The Kubebuilder Book](https://book.kubebuilder.io)
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+## Versioning, Maintenance, and Compatibility
 
-You can reach the maintainers of this project at:
+- We follow [Semantic Versioning (semver)](https://semver.org/).
+- Cluster API bootstrap provider kubeadm versioning is syncronized with
+  [Cluster API](https://github.com/kubernetes-sigs/cluster-api/blob/master/README.md).
+- The _master_ branch is where development happens, this might include breaking changes.
+- The _release-X_ branches contain stable, backward compatible code. A new _release-X_ branch
+  is created at every major (X) release.
 
-- [Slack](http://slack.k8s.io/)
-- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-dev)
+## Get involved!
+
+* Join the [Cluster API discuss forum](https://discuss.kubernetes.io/c/contributors/cluster-api).
+
+* Join the [sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
+Google Group for access to documents and calendars.
+
+* Join our Cluster API working group sessions
+  * Weekly on Wednesdays @ 10:00 PT on [Zoom][zoomMeeting]
+  * Previous meetings: \[ [notes][notes] | [recordings][recordings] \]
+
+* Provider implementer office hours
+  * Weekly on Tuesdays @ 12:00 PT ([Zoom][providerZoomMeetingTues]) and Wednesdays @ 15:00 CET ([Zoom][providerZoomMeetingWed])
+  * Previous meetings: \[ [notes][implementerNotes] \]
+
+* Chat with us on [Slack](http://slack.k8s.io/): #cluster-api
 
 ### Code of conduct
 
 Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
 
 [owners]: https://git.k8s.io/community/contributors/guide/owners.md
 [Creative Commons 4.0]: https://git.k8s.io/website/LICENSE
+
+### Useful links
+
```

Comment:
The only problem is now this sections is showing up as empty. you can reuse the footnote links here!

* [Meeting notes][notes]
* [Meeting recordings][recordings]
and so on